src/components/SettingsPanel.vue

```diff
@@ -90,7 +90,7 @@ export default {
   components: { ToggleButton },
   computed: {
     ...mapState('keymap', ['continuousInput', 'displaySizes']),
-    ...mapState('app', ['tutorialEnabled', 'darkmodeEnabled'])
+    ...mapState('app', ['tutorialEnabled', 'configuratorSettings'])
```

Comment:
this would be a good use of a getter, since you could hide the implementation details

Reply:
As the property is directly accessed and not mutated getter is not mandatory IMO. But really depends on the way you want to get the infos from the config. We can do both yes.